app/javascript/components/forms/DeleteLink.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <form :action="action" method="post" ref="form">
+    <AuthTokenInput />
+    <input type="hidden" name="_method" value="delete" />
+    <a href="#" :class="classes" v-on:click="submit"><slot /></a>
```

Comment:
Minor: i have a slight preference to put `<slot />` on its own line, to give it a little more visibility.
When scanning a component template, it's helpful to be able to quickly spot any slots defined.